WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -708,4 +653,85 @@ class PostListViewController : AbstractPostListViewController, UIViewControllerR
 
         restorePost(apost)
     }
+
+    // MARK: - Refreshing noResultsView
+
+    private func handleRefreshNoResultsView(noResultsView: WPNoResultsView) {
+        noResultsView.titleText = noResultsTitle()
+        noResultsView.messageText = noResultsMessage()
+        noResultsView.accessoryView = noResultsAccessoryView()
+        noResultsView.buttonTitle = noResultsButtonTitle()
+    }
+
+    // MARK: - NoResultsView Customizer helpers
+
+    private func noResultsAccessoryView() -> UIView {
+        if syncHelper.isSyncing {
+            animatedBox.animateAfterDelay(0.1)
+            return animatedBox
+        }
+
+        return UIImageView(image: UIImage(named: "illustration-posts"))
+    }
+
+    func noResultsButtonTitle() -> String {
+        if syncHelper.isSyncing == true || isSearching() {
+            return ""
+        }
+
+        let filterType = currentPostListFilter().filterType
+        var title: String
+
+        switch filterType {
+        case .Scheduled:
+            title = NSLocalizedString("Edit Drafts", comment: "Button title, encourages users to schedule a draft post to publish.")
+            break
+        case .Trashed:
+            title = ""
+            break
+        default:
+            title = NSLocalizedString("Start a Post", comment: "Button title, encourages users to create their first post on their blog.")
+            break
+        }
+
+        return title
```

Comment:
Same comment as above. Since each case doesn't really do much, can we switch to an inline `return` ?

BTW: *break* is not needed in Swift